setup.py

```diff
@@ -23,171 +24,178 @@
 data files.
 """
 try:
-    from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
+    from wheel.bdist_wheel import bdist_wheel
 
-    class bdist_wheel(_bdist_wheel):
+    # noinspection PyClassicStyleClass,PyAttributeOutsideInit
+    class CustomBDistWheel(bdist_wheel):
         def finalize_options(self):
-            _bdist_wheel.finalize_options(self)
+            # noinspection PyArgumentList
+            bdist_wheel.finalize_options(self)
             self.root_is_pure = False
 except ImportError:
-    bdist_wheel = None
     print("setup requires that the Python package 'wheel' be installed. "
           "Try the command 'pip install wheel'.")
     sys.exit(1)
 
 
-def getExecutableDir():
+def get_executable_dir():
     """
     Build source path on the afdko for the command-line tools.
     """
-    curSystem = platform.system()
-    if curSystem == "Windows":
-        binDir = "win"
-    elif curSystem == "Linux":
-        binDir = "linux"
-    elif curSystem == "Darwin":
-        binDir = "osx"
+    platform_system = platform.system()
+    if platform_system == "Windows":
+        bin_dir = "win"
+    elif platform_system == "Linux":
+        bin_dir = "linux"
+    elif platform_system == "Darwin":
+        bin_dir = "osx"
     else:
-        raise KeyError("Do not recognize target OS: %s" % sys.platform)
-    platform_name = get_platform()
-    return binDir, platform_name, curSystem
+        raise KeyError(
+            "Do not recognize target OS: {}".format(platform_system))
+    return bin_dir
 
 
-def compile(pgkDir):
-    if 0:
-        print("Skipping compile - hard coded.")
-        return
-    binDir, platform_name, curSystem = getExecutableDir()
-    programsDir = os.path.join(pgkDir, "Tools", "Programs")
-    if binDir == 'osx':
+def compile_package(pkg_dir):
+    bin_dir = get_executable_dir()
+    programs_dir = os.path.join(pkg_dir, "Tools", "Programs")
+    cmd = None
+    if bin_dir == 'osx':
         cmd = "sh BuildAll.sh"
-    elif binDir == 'win':
+    elif bin_dir == 'win':
         cmd = "BuildAll.cmd"
-    elif binDir == 'linux':
+    elif bin_dir == 'linux':
         cmd = "sh BuildAllLinux.sh"
-    curdir = os.getcwd()
-    subprocess.check_call(cmd, cwd=programsDir, shell=True)
-    os.chdir(curdir)
+    cur_dir = os.getcwd()
+    assert cmd, 'Unable to form command for this platform.'
+    subprocess.check_call(cmd, cwd=programs_dir, shell=True)
```

Comment:
Should this call be wrapped in a try/except so that we can listen for CalledProcessError?

Reply:
OK, done.